src/Tools/ExternalAccess/FSharp/FSharpDocumentSpan.cs

```diff
@@ -22,21 +22,12 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp
         /// <summary>
         /// Additional information attached to a document span by it creator.
         /// </summary>
-        public ImmutableDictionary<string, object> Properties { get; }
+        public ImmutableDictionary<string, object> Properties { get; } = ImmutableDictionary<string, object>.Empty;
 
         public FSharpDocumentSpan(Document document, TextSpan sourceSpan)
-            : this(document, sourceSpan, properties: null)
-        {
-        }
-
-        public FSharpDocumentSpan(
-            Document document,
-            TextSpan sourceSpan,
-            ImmutableDictionary<string, object> properties)
```

Comment:
i checked and F# never uses this constructor.  so safe to remove.  @vzarytovskii  as a heads up though.